client/reader/stream/x-post.jsx

```diff
@@ -139,10 +139,14 @@ var CrossPost = React.createClass( {
 				<SiteAndAuthorIcon
 					siteId={ this.props.post.site_ID }
 					isExternal={ this.props.post.is_external }
-					user={ post.author } />
+					user={ post.author }
+					onClick={ this.handleTitleClick }
+					href={ post.URL } />
 				<div className="reader__x-post">
 					{ post.title
-						? <h4 className="reader__post-title"><a className="reader__post-title-link" onClick={ this.handleTitleClick } href={ post.URL } target="_blank" rel="noopener noreferrer">{ xpostTitle }</a></h4>
+						? <h4 className="reader__post-title">
```

Comment:
@samouri I've update this in: https://github.com/Automattic/wp-calypso/pull/10266/commits/48e2713661b93c91d8f42ec004727d81601b24bb. Also updated `h4` to an `h1` to make it more semantic.